presto-resource-group-managers/src/test/java/com/facebook/presto/resourceGroups/db/TestResourceGroupsDao.java

```diff
@@ -117,14 +117,12 @@ private static void testSelectorUpdate(H2ResourceGroupsDao dao, Map<Long, Select
                 2,
                 Optional.of(Pattern.compile("ping.*")),
                 Optional.of(Pattern.compile("ping_source")));
-        map.remove(2);
```

Comment:
Also, the commit comment is hard to understand and needs rewording:
>Here fix is actually remove whenever .remove() with wrong type was
followed by .put() with a right one.